### PR TITLE
upgrade to work with django 2

### PR DESCRIPTION
**Pull request by relwell**
upgrade to 0.11.0, fix import issues, bandaid-fix a rendering error

this isn't perfect, and there are some autoformatting items. we also need to improve documentation significantly. but this is "good enough" for folks to utilize, and i want to make sure people are able to work around the `AppRegistryNotReady` error.

### CODE REVIEW

**Comment by belyak** [comment]
Your PR should be improved (I cannot tell how) but there is a problem:

you import within a function:

```
def autodiscover():
    from .decorators import register
    from .sites import AdminReportSite, site
    from .reports import Report
```

and if I follow README and do the next import:
```
from admin_reports import Report, register
```
I get an exception:
```
ImportError: cannot import name 'Report' from 'admin_reports'
```
because Report and register are not existing in `admin_reports.__init__` module...

**Comment by kolanos** [comment]
Closes #23

**Comment by oberix** [comment]
Thanks for the help, but I'd rather not mix stylistic corrections with actual bugfix. 
@kolanos Could you please have them in different pull requests?

Also, is that a dictionary as a default value that one on `admin_reports/views.py:31`?

File: admin_reports/views.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 class ReportList(object):
 
-    def __init__(self, request, report):
+    def __init__(self, request, report, opts={}):
```
Comment (oberix) [inline admin_reports/views.py:31]: Don't use mutables as default value for params

File: admin_reports/views.py [hunk 2]
```diff
@@ -303,7 +303,10 @@ def get_context_data(self, **kwargs):
             if form.is_valid():
                 self.report.set_params(**form.cleaned_data)
         rl = ReportList(self.request, self.report)
+        print(kwargs)
```
Comment (oberix) [inline admin_reports/views.py:306]: Please don't leave `print()` around, use `logger.debug()` instead.

**Comment by relwell** [comment]
Thanks for the feedback, but at this point I probably won't be taking this one to the finish line. I have a workaround for where I'm using it, and my use case is most likely to change to where I don't need this library at all.

**Comment by oberix** [comment]
To everyone discussing this topic: I did not see so far a pull request that actually even try to address the issue of making this module work with python3/django2. I don't have the time nor the will at the moment to take care of this myself. I'd appreciate any contribution that is well though and straight to the point though (it shouldn't be too hard as far as I know from other tasks like this). At the moment I'm not using this module anymore, that doesn't mean that I'm leaving the development for good, it just means that I can't say when I will have the time to come back to it and make a new release.

**Pull request closed without merging**